projects/rust-coreutils/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Sylvestre Ledru <sylvestre@debian.org>
```

Comment:
It has to be `# Copyright 2023 Google LLC` for us to accept this.
If youre uncomfortable with this, you can put the real content of build.sh in a script in the original repo and use build.sh here to call into that script.

Reply:
oh ok :) fun
no worries, I don't care about this :)

Reply:
fixed 
btw, the bot could give more helpful error messages :)